domain-management/src/main/java/org/jboss/as/domain/management/access/ApplicationClassificationConfigResourceDefinition.java

```diff
@@ -121,17 +119,29 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
 
         @Override
         public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
-            final String attribute = operation.require(NAME).asString();
+            PathAddress address = PathAddress.pathAddress(operation.get(OP_ADDR));
```

Comment:
If you need the address of the current op, use 'context.getCurrentAddress()'.

I don't think you need it though; see my next comment.